src/components/Buttons/MainButton/MainButton.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import '../Button.scss';
+
+type Props = {
+  text: string,
+  where: string,
+  className: string,
+};
+
+export const MainButton: React.FC<Props> = ({
+  text,
+  where,
+  className,
+}) => (
+  <Link
+    to={where}
+    className={`button button--main ${className}`}
```

Comment:
better to use `classNames` package

Reply:
not fixed
https://github.com/mate-academy/react_phone-catalog/pull/191/files#r1271265569